packages/@sfx/sayt-driver-plugin/src/sayt-driver-plugin.ts

```diff
@@ -136,6 +137,7 @@ export default class SaytDriverPlugin implements Plugin {
  */
 export interface AutocompleteRequestConfig extends QueryTimeAutocompleteConfig {
   query: string;
+  config: any;
```

Comment:
`config` is the `QueryTimeAutocompleteConfig`.  This interface should be changed to no longer extend `QueryTimeAutocompleteConfig` and `config` should be of that type.

Reply:
`config` should also be optional.